__tests__/integration.tests.js

```diff
@@ -67,3 +67,38 @@ describe('ALL /notapath', () => {
             });
     });
 });
+describe("/api/articles/:article_id", () => {
+    test('GET: 200, should respond with an array containing a single object with given id and correct keys', () => {
+        return request(app)
+            .get("/api/articles/2")
+            .expect(200)
+            .then((response) => {
+                const article = response.body;
+                expect(article).toHaveProperty("author");
+                expect(article).toHaveProperty("title");
+                expect(article).toHaveProperty("article_id");
```

Comment:
Please check the actual article id here, to make sure it's 2 as well, so we've received the right one.

Reply:
expect(article).toHaveProperty("article_id", 2);